### PR TITLE
Tailwind and Intial folder structure setup

### PR DESCRIPTION
**Pull request by Arvindjitta**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,18 +11,23 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@heroicons/react": "^2.1.3",
```
Comment (vikas62081) [inline package.json:14]: remove this

File: package.json [hunk 2]
```diff
@@ -11,18 +11,23 @@
...
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.23.0"
   },
   "devDependencies": {
     "@types/react": "^18.2.66",
     "@types/react-dom": "^18.2.22",
     "@typescript-eslint/eslint-plugin": "^7.2.0",
     "@typescript-eslint/parser": "^7.2.0",
     "@vitejs/plugin-react": "^4.2.1",
+    "autoprefixer": "^10.4.19",
```
Comment (vikas62081) [inline package.json:25]: autoprefixer, postcss might not required

File: postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export default {
```
Comment (vikas62081) [inline postcss.config.js:1]: might not required, as we not required postcss

File: src/App.tsx [hunk 4]
```diff
@@ -1,35 +1,22 @@
...
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
-}
+    <Router>
+      <Layout>
+        <Routes>
+          <Route path="/" element={<HomePage />} />
```
Comment (vikas62081) [inline src/App.tsx:13]: create a different component called RouteProvider and move this code there

File: src/components/BottomNav.tsx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+// src/components/BottomNav.tsx
+import React from "react";
+
+const BottomNav: React.FC = () => {
+  return (
+    <div className=" bg-gray-800 text-white p-4 fixed inset-x-0 bottom-0 flex justify-around">
+      <button className="focus:outline-none">Home</button>
```
Comment (vikas62081) [inline src/components/BottomNav.tsx:7]: use map to render multiple buttons

File: src/components/layout/Layout.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+// src/components/layout/Layout.tsx
+import React, { ReactNode } from "react";
+
```
Comment (vikas62081) [inline src/components/layout/Layout.tsx:3]: better have have main file as index

**Review by vikas62081 — CHANGES_REQUESTED** [review]
component name have typo, rename it
 src/components/common/Buttos.tsx

File: package.json (hunk 1 above)
Comment (Arvindjitta) [reply package.json:14]: removed

File: package.json (hunk 2 above)
Comment (Arvindjitta) [reply package.json:25]: removing this caused ui break.. so i installed it back, we shall look in it later

File: postcss.config.js (hunk 3 above)
Comment (Arvindjitta) [reply postcss.config.js:1]: removing it caused issue, so i'm going with old one.

File: src/App.tsx (hunk 4 above)
Comment (Arvindjitta) [reply src/App.tsx:13]: I have made a separate component with name "AppRouter". As there is one "RouterProvider" from react-route-dom is similar to the propsed name "RouteProvider".

**Review by vikas62081 — APPROVED** [review]

File: src/components/BottomNav.tsx (hunk 5 above)
Comment (Arvindjitta) [reply src/components/BottomNav.tsx:7]: completed

File: src/components/layout/Layout.tsx (hunk 6 above)
Comment (Arvindjitta) [reply src/components/layout/Layout.tsx:3]: did that as well

**Pull request merged**